src/static/helpers/slasHelper.ts

```diff
@@ -426,6 +454,10 @@ export async function loginRegisteredUserB2C(
     redirectURI: string;
     usid?: string;
     dnt?: boolean;
+  } & {[key in `c_${string}`]: any},
+  opts?: {
+    headers?: {[key: string]: string};
+    body?: {[key in `c_${string}`]: any} & LoginRequest;
```

Comment:
Reason we allow body here is the `authenticateCustomer` [API](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-login?meta=authenticateCustomer) nows allows a custom body that has pattern ^c_.+$. We should allow that to go through slas helper here.